content/docs/procedures/form-new-working-group.md

```diff
@@ -93,7 +93,7 @@ The basic steps to setup a working group are as follows:
     README][README]. Feel free to put N/A for any column that doesn't
     seem to apply.
   - Add a subdirectory to the [`working-groups`] directory. You do
-    this by copying the [`.template`][template] directory to a
+    this by copying the [`template`]({{< relref "/docs/working-groups/template/_index.md" >}}) directory to a
     directory of your own and modifying its contents as appropriate.
     Feel free to remove sections if they don't seem to apply: many
     working groups start out with relatively little structure but
```

Comment:
There's a broken link a few lines down from this (that GitHub won't let me comment on) - `#t-compiler` should be wrapped in `[]`.

Reply:
David, we fixed the links in this page but there is one that we could not fix, the ```compiler-team meeting check-in list``` goes to  https://github.com/rust-lang/compiler-team/blob/master/procedures/check_in but there is a 404 error in the directory.

Reply:
That should have linked to [this page](https://github.com/rust-lang/compiler-team/blob/master/about/triage-meeting.md#working-group-check-in).

Reply:
There are still broken links on line 20 and line 96.

Reply:
Fixed and making sure there are no bronken links on any file